### PR TITLE
[MIG] l10n_eu_nace: Migration to 17.0

### PR DESCRIPTION
**Pull request by manuelregidor**
Standard migration

T-6028

### CODE REVIEW

**Comment by manuelregidor** [comment]
@HaraldPanten @ValentinVinagre @luis-ron

**Review by luis-ron — APPROVED** [review]
Functional review: LGTM 👍🏻

**Review by HaraldPanten — APPROVED** [review]
Functional review

**Comment by HaraldPanten** [comment]
@etobella @pedrobaeza could you give us a hand with this PR, in order to move it forward?

THX!

File: l10n_eu_nace/tests/test_en_nace_request_results.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def create_response(*lists):
     response = Response()
     response.code = "200"
     response.status_code = 200
-    merged_lists = list(zip(*lists))
+    merged_lists = list(zip(*lists, strict=False))
```
Comment (ValentinVinagre) [inline l10n_eu_nace/tests/test_en_nace_request_results.py:140]: The `strict=False ` would not be necessary. By default it is already in the library.
https://peps.python.org/pep-0618/#add-itertools-zip-strict

**Review by ValentinVinagre — CHANGES_REQUESTED** [review]
Some changes

**Review by etobella — APPROVED** [review]
/ocabot migration l10n_eu_nace

Technical review

File: l10n_eu_nace/tests/test_en_nace_request_results.py (hunk 1 above)
Comment (manuelregidor) [reply l10n_eu_nace/tests/test_en_nace_request_results.py:140]: Giving a value to the "strict" parameter is required by pre-commit.

Comment (ValentinVinagre) [reply l10n_eu_nace/tests/test_en_nace_request_results.py:140]: aps... @pedrobaeza @etobella do u know why is required in pre-commit? 😄

Comment (pedrobaeza) [reply l10n_eu_nace/tests/test_en_nace_request_results.py:140]: No idea, but it's not a big deal IMO

Comment (ValentinVinagre) [reply l10n_eu_nace/tests/test_en_nace_request_results.py:140]: It's more curiosity haha thanks :)

**Pull request merged**